### PR TITLE
Fixes cigarettes runtiming when preferences generation attempts to light them before initialization is done.

### PR DESCRIPTION
**Pull request by silicons**
image

### CODE REVIEW

File: code/game/objects/items/cigs_lighters.dm [hunk 1]
```diff
@@ -155,7 +155,7 @@ CIGARETTE PACKETS ARE IN FANCY.DM
 
 
 /obj/item/clothing/mask/cigarette/proc/light(flavor_text = null)
-	if(lit)
+	if(lit || !initialized)
 		return
```
Comment (Cyberboss) [inline code/game/objects/items/cigs_lighters.dm]: if(!initialized)
    icon_state = icon_on
    return

**Review by Cyberboss — CHANGES_REQUESTED** [review]

**Comment by optimumtact** [comment]
honestly things shouldn't process before intialization @tgstation/commit-access  @MrStonedOne thoughts?

**Comment by AnturK** [comment]
Outfits have a parameter for these, it should be changed at that level.

**Comment by optimumtact** [comment]
yeah but I'm saying all process tickers shouldn't even tick until SSObjs is done intialising

**Comment by AnturK** [comment]
But that's the case already ? I'm confused.

**Comment by MrStonedOne** [comment]
oranges is setting `process link(...)` and thinking they are `process()` calls because he forgot byond converts `_` to a space meaning the caller is actually `process_link`

**Review by Cyberboss — APPROVED** [review]

**Comment by AnturK** [comment]
Why i even bother commenting.

**Pull request merged**